src/views/contact.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends "layout.html.twig" %}
```

Comment:
In silex projects I generally quite like this as twig variable although it slightly violates MVC

```
$app->before(function () use ($app) {
    $app['twig']->addGlobal('layout', $app['twig']->loadTemplate('base.html.twig'));
});
```